Packs/SuspiciousDomainHunting/Layouts/layoutscontainer-Suspicious_Domain_Hunting_Incident_Layout.json

```diff
@@ -0,0 +1,1592 @@
+                                "fieldId": "closereason",
+                                "height": 22,
+                                "id": "incident-closeReason-field",
+                                "index": 1,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "closenotes",
+                                "height": 22,
+                                "id": "incident-closeNotes-field",
+                                "index": 2,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Closing Information",
+                        "static": false,
+                        "w": 1,
+                        "x": 1,
+                        "y": 2
+                    },
+                    {
+                        "displayType": "ROW",
+                        "h": 2,
+                        "hideName": false,
+                        "i": "caseinfoid-jk8ghehhzw-field-changed-caseinfoid-field-changed-caseinfoid-f7d69960-78c9-11ee-825c-270ac8ef585b",
+                        "items": [
+                            {
+                                "args": {
+                                    "closeNotes": {
+                                        "simple": "False Positive"
+                                    },
+                                    "closeReason": {
+                                        "simple": "False Positive"
+                                    }
+                                },
+                                "buttonClass": "success",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "fc261d60-78c9-11ee-825c-270ac8ef585b",
+                                "index": 0,
+                                "name": "Resolve As False Positive",
+                                "scriptId": "Builtin|||closeInvestigation",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            },
+                            {
+                                "args": {},
+                                "buttonClass": "warning",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "150cc770-78ca-11ee-825c-270ac8ef585b",
+                                "index": 1,
+                                "name": "Run !Domain",
```

Comment:
Let's change the button color to blue or something more neutral. I believe enrichment buttons don't necessarily need to be in orange.